selfdrive/camerad/cameras/camera_qcom2.cc

```diff
@@ -76,12 +76,29 @@ int cam_control(int fd, int op_code, void *handle, int size) {
   return ret;
 }
 
+int32_t device_acquire(int fd, int32_t session_handle, void *data) {
```

Comment:
return `std::optional<int32_t>` and let the calling site assert on error condition.